index.bs

```diff
@@ -297,13 +305,13 @@ spec: webidl
           the UA receives <a>new information about the user's intent</a>.
         </li>
         <li>
-          Ask the user to choose one of the options or deny permission, and wait
-          for them to choose. If the calling algorithm specified extra
-          information to include in the prompt, include it.
+          Wait for the user to choose either one of the options or deny permission.
```

Comment:
We could wait a very long time if we wait for them to choose something we haven't asked them about. ;)